src/scripts/pokedex/PokedexHelper.ts

```diff
@@ -3,6 +3,16 @@ import TypeColor = GameConstants.TypeColor;
 class PokedexHelper {
     public static toggleStatisticShiny = ko.observable(true);
     public static hideShinyImages = ko.observable(false);
+    public static filters = {
```

Comment:
Is it possible to just bind to the setting, instead of having this object?
Then the filters should be saved too, right?

Reply:
:+1: